src/main/java/seedu/bookmark/logic/parser/CliSyntax.java

```diff
@@ -9,7 +9,7 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_GENRE = new Prefix("g/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-    public static final Prefix PREFIX_TOTAL_PAGES = new Prefix("tp/");
-    public static final Prefix PREFIX_BOOKMARK = new Prefix("b/");
+    public static final Prefix PREFIX_TOTAL_PAGES = new Prefix("total/");
+    public static final Prefix PREFIX_BOOKMARK = new Prefix("bookmark/");
```

Comment:
According to the user guide, the prefix should be "tp/" for total pages and "b/" for bookmark for consistency purposes right? Maybe we can discuss this further!

Reply:
Looks like there are some discrepancies between our UG and DG. Let's go over this again in the next meeting. In the meantime, I'll make the relevant changes! Thanks!